cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -350,6 +355,10 @@ private void reconcileStorageSegment(DebugStreamSegmentContainer container) thro
         for (String segment : segments) {
             List<TableEntry> entries = extension.get(NameUtils.getStorageMetadataSegmentName(container.getId()), Collections.singletonList(BufferView.wrap(segment.getBytes(StandardCharsets.UTF_8))), TIMEOUT).get();
             TableEntry entry = entries.get(0);
```

Comment:
Can **entries** be empty in any case?

Reply:
or `null` :)

Reply:
no. if the key doesnt exist or the value is null, we put in a `null` as an item in the list .